docs/standard_library/cryptographic_primitives/00_hashes.mdx

```diff
@@ -137,3 +137,13 @@ fn main() {
     let hash = std::hash::mimc_bn254(x);
 }
 ```
+
+## hash_to_field
+
+```rust
+fn hash_to_field<N>(_input : [Field; N]) -> Field {}
+```
+
+See the [Merkle Proof Membership](../../examples/merkle-proof) page for an example.
```

Comment:
This isn't used in `compute_merkle_root` as that uses Pedersen, `hash_to_field(input)` is actually defined along the lines of `blake2s(input) as Field` (psuedocode).

Essentially we're taking the 256bit hash value from blake2s and then doing `hash % <field_modulus>` so that it will fit in a `Field`.

This suggestion is probably not as readable as it should be but it's late.
```suggestion
Calculates the `blake2s` hash of the inputs and returns the hash modulo the field modulus to return a value which can be represented as a `Field`
```

Reply:
lgtm! :pray: